build_config/lib/src/key_normalization.dart

```diff
@@ -41,11 +41,17 @@ String normalizeTargetKeyDefinition(String targetKey, String packageName) =>
 ///
 /// Example normalizations:
 ///
+///   - "$default" => "$packageName:$packageName"
+///   - "$default:$default": => "$packageName:$packageName"
 ///   - "some_package" => "some_package:some_package"
 ///   - ":some_target" => "$packageName:some_target"
 ///   - "some_package:some_target" => "some_package:some_target"
 String normalizeTargetKeyUsage(String targetKey, String packageName) =>
-    _normalizeUsage(targetKey, packageName, ':');
+    targetKey == _defaultTargetNamePlaceholder ||
+            targetKey ==
+                '$_defaultTargetNamePlaceholder:$_defaultTargetNamePlaceholder'
```

Comment:
might not be necessary, but do we want to accept `:$default`? We do accept `:my_package` I think

Reply:
PTAL again, thanks @natebosch